config/reports/virusScanReport.js

```diff
@@ -0,0 +1,47 @@
+import { generateReportFromData } from '../helpers.js';
+import { querySudo as query } from '@lblod/mu-auth-sudo';
+
+export default {
+  cronPattern: '4 2 * * *',
+  name: 'virusScanReport',
+  execute: async () => {
+    const reportData = {
+      title: 'Virus Scan Report',
+      description:
+        'A list of files that are considered malicious, or where the status is unknown',
+      filePrefix: 'virus-scan-report',
+    };
+
+    const queryString = `
+PREFIX stix: <http://docs.oasis-open.org/cti/ns/stix#>
+PREFIX nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#>
+PREFIX nie: <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#>
+
+SELECT DISTINCT ?file ?fileName ?scanResult ?scanDate WHERE {
+  GRAPH ?g {
+    ?analysis a stix:MalwareAnalysis ;
+      stix:analysis_started ?scanDate ;
+      stix:sample_ref ?file ;
+      stix:result ?scanResult .
+      FILTER (?scanResult IN ("malicious", "unknown"))
+
+      ?file nfo:fileName ?fileName .
+  }
+}
+ORDER BY DESC(?scanDate)`;
```

Comment:
Small indentation nitpick; I'm not sure if Github will display this properly, but in case it doesn't, I meant something akid to [this](https://github.com/lblod/app-digitaal-loket/blob/master/config/reports/submissionsReport.js#L21-L41).

```suggestion
    const queryString = `
      PREFIX stix: <http://docs.oasis-open.org/cti/ns/stix#>
      PREFIX nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#>
      PREFIX nie: <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#>
      
      SELECT DISTINCT ?file ?fileName ?scanResult ?scanDate WHERE {
        GRAPH ?g {
          ?analysis a stix:MalwareAnalysis ;
            stix:analysis_started ?scanDate ;
            stix:sample_ref ?file ;
            stix:result ?scanResult .
            FILTER (?scanResult IN ("malicious", "unknown"))
      
            ?file nfo:fileName ?fileName .
        }
      }
      ORDER BY DESC(?scanDate)
    `;
```

Reply:
updated the indentation